ui/components/app-wrapper.js

```diff
@@ -22,10 +22,17 @@ Router.onRouteChangeStart = NProgress.start;
 export default function wrapPage(Page, dataFn) {
   function WrappedPage(props) {
     if (props.err) return <ErrorPage err={props.err} />;
-    return <HeaderFooter><Page {...props} /></HeaderFooter>;
+    return <HeaderFooter showSearch={ props.url.pathname !== "/" }><Page {...props} /></HeaderFooter>;
```

Comment:
Do you think we can get by without testing for global state? It's difficult to test and tends to be fragile.

Maybe we could change the function to `wrapPage(Page, dataFn, headerFooterProps)` and then have `Homepage` pass in `{ showSearch: false }`?

Reply:
I agree. I like this. Making an update.